src/schema/v2/me/index.ts

```diff
@@ -578,6 +578,7 @@ const MeField: GraphQLFieldConfig<void, ResolverContext> = {
       "showsByFollowedArtists",
       "newWorksFromGalleriesYouFollowConnection",
       "myCollectionInfo",
+      "recentlyViewedArtworkIds",
```

Comment:
Is this a field that's accessible in the schema?

Reply:
yes, this is the field of the me

Reply:
Right, but this section is about fields in the MP schema, not fields in the `me` JSON.